### PR TITLE
Revert "Revert "Update to latest api explorer, remove unused files""

### PR DESCRIPTION
**Pull request by mlee-asana**
Make the API explorer updates without removing the popup

### CODE REVIEW

File: source/javascripts/explorer/asana-explorer.js [hunk 1]
```diff
@@ -667,7 +667,7 @@ var Explorer = /** @class */ (function (_super) {
 })(Explorer || (Explorer = {}));
```
Comment (jv-asana) [inline source/javascripts/explorer/asana-explorer.js:667]: Do we still need this file if it's not being used? Was this file auto generated?

Comment (mlee-asana) [reply source/javascripts/explorer/asana-explorer.js:667]: it's being used in `popup_receiver.html`: https://github.com/Asana/developer-docs/blob/master/source/explorer/popup_receiver.html#L5

i'll switch it to use the min version too and then i think we can just get rid of this file

Comment (jv-asana) [reply source/javascripts/explorer/asana-explorer.js:667]: I guess it's still being referenced in `popup_receive.html` https://github.com/Asana/developer-docs/blob/master/source/explorer/popup_receiver.html#L5

**Review by jv-asana — APPROVED** [review]
LGTM

**Pull request merged**